README.md

```diff
@@ -138,8 +155,8 @@ Now that you are collecting metrics for machines hosting applications and their
 
 ## Related Sandbox
 
```

Comment:
How about "Related DevNet Sandbox" for the heading?

Reply:
Yes, Anne. Good to add DevNet to the title, so that it will not be very generic.
Changed as suggested!